low-level/files/opt/seagate/sspl/conf/sspl.prepare.conf.tmpl

```diff
@@ -0,0 +1,26 @@
+server_node:
+  30512e5ae6df9f1ea02327bab45e499d:
+    bmc:
```

Comment:
Missing
>server_node>{machine-id}>network>management>interfaces
server_node>{machine-id}>network>management>public_fqdn
server_node>{machine-id}>network>data>public_interfaces
server_node>{machine-id}>network>data>public_fqdn
server_node>{machine-id}>network>data>private_interfaces
server_node>{machine-id}>network>data>private_fqdn

And refer latest keys from LR R2 provisioning sheet

Reply:
Added them.
Why did we include fqdn ?

Reply:
FQDN instead of IPs, to check their accessibility status